pkg/controller/astarte/reconcile/utils.go

```diff
@@ -390,9 +390,14 @@ func getImageForClusteredResource(defaultImageName, defaultImageTag string, reso
 	return image
 }
 
-func getDeploymentStrategyForClusteredResource(cr *apiv1alpha1.Astarte, resource apiv1alpha1.AstarteGenericClusteredResource) appsv1.DeploymentStrategy {
+func getDeploymentStrategyForClusteredResource(cr *apiv1alpha1.Astarte, resource apiv1alpha1.AstarteGenericClusteredResource, component apiv1alpha1.AstarteComponent) *appsv1.DeploymentStrategy {
 	if resource.DeploymentStrategy != nil {
-		return *resource.DeploymentStrategy
+		return resource.DeploymentStrategy
+	}
+	if component == apiv1alpha1.DataUpdaterPlant && resource.DeploymentStrategy == nil {
```

Comment:
So Flow shouldn't be part of this strategy?

Reply:
Flow was introduced since 1.0. It will be handled in a dedicated PR targeting 1.0.